LSP.sublime-settings

```diff
@@ -134,6 +134,9 @@
   // "phantom" - show a phantom on the top when code actions are available
   "show_code_lens": "annotation",
 
+   // Show inlay hints
```

Comment:
It's quite redundant comment.
Maybe something like this at least?
`// Show inlay hints that interleave with the code.`